openmdao/core/vec_wrapper.py

```diff
@@ -278,7 +278,7 @@ def _get_scalar_units_complex(self):
 
     def _set_arr(self, value):
         """Set an array value."""
-        self.val[:] = value.flat
+        self.val[:] = value.real.flat
```

Comment:
because, when you do complex_step the returns are complex. so at some point, it tries to use this function to set the result (after CS is done maybe? I didn't trace it through to see who was calling). 

When this gets used after CS is done, you get a warning about it discarding the imag part. I don't want that warning confusing users, so I just manually discarded it. we've been getting that warning for a while now...